examples/multipro/node.go

```diff
@@ -35,15 +35,15 @@ func NewNode(host host.Host, done chan bool) *Node {
 }
 
 // Authenticate incoming p2p message
-// message: a protobufs go data object
+// message: a protobuf go data object
 // data: common p2p message data
 func (n *Node) authenticateMessage(message proto.Message, data *p2p.MessageData) bool {
 	// store a temp ref to signature and remove it from message data
 	// sign is a string to allow easy reset to zero-value (empty string)
 	sign := data.Sign
 	data.Sign = nil
 
-	// marshall data without the signature to protobufs3 binary format
+	// marshall data without the signature to protobuf binary format
```

Comment:
haven't looked but if it's actually marshaling to proto3 we may want to keep that distinction

Reply:
fixed